rustls/src/crypto/mod.rs

```diff
@@ -493,6 +508,19 @@ pub trait ActiveKeyExchange: Send + Sync {
 
     /// Return the group being used.
     fn group(&self) -> NamedGroup;
+
+    /// FFDHE group the `ActiveKeyExchange` is operating in.
+    ///
+    /// Return `None` if this group is not a FFDHE one.
+    ///
+    /// The default implementation returns `None`, so this function _must_ be provided
+    /// for FFDHE groups.
+    ///
+    /// `rustls::ffdhe_groups` contains suitable values to return from this,
+    /// for example [`rustls::ffdhe_groups::FFDHE2048`][crate::ffdhe_groups::FFDHE2048].
+    fn ffdhe_group(&self) -> Option<FfdheGroup<'static>> {
```

Comment:
Nit: maybe sort this before `pub_key()`?

Reply:
I feel like this should live alongside `group` (different representation of the same data) so I'm minded to leave this one here.